tests/images.bats

```diff
@@ -157,3 +157,9 @@ load helpers
   buildah rm -a
   buildah rmi -a -f
 }
+
+@test "image digest test" {
+  buildah pull busybox
```

Comment:
`run_buildah`

Reply:
This is just a prepare step. If I understand the difference between run_buildah and buildah, we should use run_buildah in the test steps, but use buildah in the prepare steps.

Reply:
One much more important difference is that `run_buildah` will _display the command being run_, and its output, and will check its exit status. If `buildah pull` fails (probably because of a network flake), BATS will exit with a hard-to-debug error. `run_buildah` will give much better information.